src/ParticipantManagementService/GetParticipant/local.settings.json

```diff
@@ -0,0 +1,10 @@
+{
+  "IsEncrypted": false,
+  "Values": {
+    "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+    "FUNCTIONS_WORKER_RUNTIME": "dotnet"
```

Comment:
Let's use the "dotnet-isolated" runtime for this function. It will also require changes in the GetParticipant.csproj and GetParticipant.cs. Take a look at `EpisodeManagementService/CreateUpdateEpisode` for an example of a function using the isolated worker model.